src/svc/cloud-data-sources/paths/orgs-orgID-datasources.yml

```diff
@@ -0,0 +1,40 @@
+    get:
+      summary: ''
+      operationId: get-orgs-orgID-datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: ./svc/cloud-data-sources/schemas/Datasource.yml
+      description: Get a list of datasources for this organization
+    post:
+      summary: ''
+      operationId: post-orgs-orgID-datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                  endpoint:
+                    $ref: '../schemas/ListenerEndpoint'
+      description: Create a datasource with a specific plugin configuration
+      requestBody:
+        description: ''
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                config:
+                  type: object
```

Comment:
does this object type have a specific definition?

Reply:
yes, but it's dynamic and I would think outside the scope of the api definition. basically the sum of all possible configs for all (supported) telegraf plugins (as json)